components/UseCaseCard.vue

```diff
@@ -22,13 +35,37 @@
 
 export default {
   props: {
-    panel: {
+    title: {
+      type: String,
+      default: null,
+      required: false
+    },
+    description: {
+      type: String,
+      default: null,
+      required: false
+    },
+    usecase: {
       type: Array,
       default: null,
       required: false
     }
-  },
-  mounted () {
   }
 }
 </script>
+
+<style>
+.vueperslide__image {
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 300px;
+  width: auto;
+}
+.vueperslide__title {
+  font-size: 1.5em;
+  opacity: 0.75;
+  background-color: rgb(255, 255, 255);
+  border-radius: 12px;
+  padding: 12px;
+}
```

Comment:
Since we don't show the title anymore, I guess we don't have to style it either, or do we?

Reply:
Yep! Thanks for pointing this out. I will remove them.